### PR TITLE
chore: Polish code

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: src/abstract/CSBondCore.sol [hunk 1]
```diff
@@ -153,7 +153,9 @@ abstract contract CSBondCore is ICSBondCore {
         uint256 sharesBefore = LIDO.sharesOf(address(this));
         WSTETH.unwrap(amount);
         uint256 sharesAfter = LIDO.sharesOf(address(this));
-        _increaseBond(nodeOperatorId, sharesAfter - sharesBefore);
+        unchecked {
```
Comment (vgorkavenko) [inline src/abstract/CSBondCore.sol:156]: it has `unchecked` internally (and `_unsafeReduceBond` as well)

Comment (madlabman) [reply src/abstract/CSBondCore.sol:156]: `unchecked` is being applied syntactically. So it has nothing to do with the internal `unchecked`s.

File: src/abstract/CSBondLock.sol [hunk 2]
```diff
@@ -120,11 +122,13 @@ abstract contract CSBondLock is ICSBondLock, Initializable {
         if (blocked < amount) {
             revert InvalidBondLockAmount();
         }
-        _changeBondLock(
-            nodeOperatorId,
-            $.bondLock[nodeOperatorId].amount - amount,
-            $.bondLock[nodeOperatorId].retentionUntil
-        );
+        unchecked {
```
Comment (madlabman) [inline src/abstract/CSBondLock.sol:125]: // NOTE: It's checked via `getActualLockedBond` call.

**Review by skhomuti — APPROVED** [review]

File: src/abstract/CSBondLock.sol (hunk 2 above)
Comment (dgusakov) [reply src/abstract/CSBondLock.sol:126]: Done

**Review by vgorkavenko — APPROVED** [review]

**Review by madlabman — APPROVED** [review]

**Pull request merged**